mod-infra/src/main/scala/at/forsyte/apalache/infra/passes/SourceOption.scala

```diff
@@ -0,0 +1,14 @@
+package at.forsyte.apalache.infra.passes
+
+/** Defines the data sources supported in the [[PassOptions]] */
+object SourceOption {
+
+  /** Supported data sources */
+  sealed trait T
+
+  /** Data to be loaded from a file */
+  case class File(file: java.io.File) extends T
+
+  /** Data supplied as a string */
+  case class String(content: java.lang.String) extends T
```

Comment:
Should we call this
```suggestion
  case class TlaString(content: java.lang.String) extends T
```

to match the method names in `SanyParserPassImpl`?

Reply:
This could be used for strings that represent any kind of data, not just TLA, so I think that would be an overy specific name for the class.